src/main/java/rx/schedulers/Schedulers.java

```diff
@@ -32,7 +35,14 @@
     private final Scheduler ioScheduler;
     private final Scheduler newThreadScheduler;
 
-    private static final Schedulers INSTANCE = new Schedulers();
+    private static Schedulers INSTANCE;
+
+    private static synchronized Schedulers getInstance() {
+        if (INSTANCE == null) {
+            INSTANCE = new Schedulers();
+        }
+        return INSTANCE;
+    }
```

Comment:
Synchronized holds the potential of getting blocked if somebody else is already in it. If the Schedulers is not constantly and concurrently reset all the time, the atomic approach is a single non-blocking volatile read of a non-null instance value.